aries-site/src/examples/templates/page-layouts/BreakPointsTable.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+import {
+  Box,
+  Table,
+  TableCell,
+  TableHeader,
+  TableBody,
+  TableRow,
+  Text,
+} from 'grommet';
+
+const data = [
+  {
+    id: 1,
+    size: (
+      <Text color="text-strong" weight="bold">
+        XSmall
+      </Text>
+    ),
+    range: <Text>576 and below</Text>,
+    margin: <Text>12</Text>,
+  },
+  {
+    id: 2,
+    size: (
+      <Text color="text-strong" weight="bold">
+        Small
+      </Text>
+    ),
+    range: <Text>577 - 768</Text>,
+    margin: <Text>24</Text>,
+  },
+  {
+    id: 3,
+    size: (
+      <Text color="text-strong" weight="bold">
+        Medium
+      </Text>
+    ),
+    range: <Text>769 - 1080</Text>,
+    margin: <Text>24</Text>,
+  },
+  {
+    id: 4,
+    size: (
+      <Text color="text-strong" weight="bold">
+        Large
+      </Text>
+    ),
+    range: <Text>1081 - 1439</Text>,
+    margin: <Text>48</Text>,
+  },
+  {
+    id: 5,
+    size: (
+      <Text color="text-strong" weight="bold">
+        XLarge
+      </Text>
+    ),
+    range: <Text>1440 and above</Text>,
+    margin: <Text>48</Text>,
+  },
+];
+
+const columns = [
+  {
+    property: 'size',
+    label: 'Size',
+  },
+  {
+    property: 'range',
+    label: 'Range',
+  },
+  {
+    property: 'margin',
+    label: 'Margin',
+  },
+];
+
+export const BreakPointsTable = () => (
+  <Box pad="small" align="start">
```

Comment:
Can we style this table to match our DataTable styling?

Reply:
yes let me look at that

Reply:
Possible that we can just do in markdown similar to:

```
| T-shirt Size | Dimension (pixels) |
| ------------ | -----------------: |
| xxsmall      | 48   |
| xsmall       | 96   |  
| small        | 192  |  
| medium       | 384  |  
| large        | 768  |  
| xlarge       | 1152 |  
| xxlarge      | 1536 |  
| full         | 100% |  
```

| T-shirt Size | Dimension (pixels) |
| ------------ | -----------------: |
| xxsmall      | 48   |
| xsmall       | 96   |  
| small        | 192  |  
| medium       | 384  |  
| large        | 768  |  
| xlarge       | 1152 |  
| xxlarge      | 1536 |  
| full         | 100% |

Reply:
good idea Ill do that now